src/wallet/wallet2.cpp

```diff
@@ -4685,28 +4728,24 @@ void wallet2::light_wallet_get_address_txs()
     }
 
     // Do not add tx if empty. 
-    if(total_sent == 0 && total_received == 0)
+    if(total_sent + total_received == 0)
```

Comment:
Why ? Even though unlikely, the addition could wrap, so there seems to be no advantage, but one unlikely cons.